### ISSUE TITLE
TypeError: unorderable types: NoneType() < int()

### ISSUE THREAD

**Issue by iperov** [deepfakes/faceswap#273]
@torzdf
you didnt add backward compatibility in your image rotating system
```
Failed to convert image: E:\FaceSwap\workspace\data_dst\out00132.png. Reason: un
orderable types: NoneType() < int()
Traceback (most recent call last):
  File "E:\FaceSwap\_internal\bin\faceswap\scripts\convert.py", line 225, in con
vert
    for idx, face in faces:
  File "E:\FaceSwap\_internal\bin\faceswap\lib\cli.py", line 146, in get_faces_a
lignments
    if face.r != 0: image = rotate_image(image, face.r)
  File "E:\FaceSwap\_internal\bin\faceswap\lib\utils.py", line 37, in rotate_ima
ge
    if angle < 0: angle = sum((360, angle))
TypeError: unorderable types: NoneType() < int()
```

**Comment by iperov** [comment]
suggest fix
```
class DetectedFace(object):
    def __init__(self, image=None, r=None, x=None, w=None, y=None, h=None, landmarksXY=None):
        self.image = image
        if r is None: r = 0
```

**Comment by Clorr** [comment]
We pushed a fix yesterday in #269 , isn't it working?

**Comment by iperov** [comment]
ok close
